tests/test_bitstruct.py

```diff
@@ -215,6 +215,11 @@ def test_pack_unpack(self):
         unpacked = unpack('f64', packed)
         self.assertEqual(unpacked, (1.0, ))
 
+        if sys.version_info[0] >= 3 and sys.version_info[1] >= 5:
```

Comment:
Half precision was introduced in Python 3.6. Also, change to `if sys.version_info >= (3, 6)`. It works perfectly fine to compare tuples of digits in Python.

Reply:
Done. Thanks!